container.go

```diff
@@ -101,28 +102,29 @@ type ContainerRequest struct {
 	Cmd             []string
 	Labels          map[string]string
 	Mounts          ContainerMounts
-	Tmpfs           map[string]string
+	Tmpfs           map[string]string // Deprecated: Use PreCreationHook instead
 	RegistryCred    string
 	WaitingFor      wait.Strategy
 	Name            string // for specifying container name
```

Comment:
I would deprecate this, risk for collisions and generally not needed in the context of TC usage.

Reply:
Indeed. But I'd do it as part of a new PR, to make it even clearer that passing a name could be seen as an antipattern from testing standpoint